README.md

```diff
@@ -39,6 +40,11 @@ Eventually these tests will be replaced with BMO's own
 [e2e tests](test/e2e/README.md). BMO e2e tests are currently under active
 development. Here are the tests which run on different branches in BMO:
 
+**E2e tests:**
+
+* **/test metal3-bmo-e2e-test-pull** runs required BMO e2e tests on your PR. Works for main and release-0.5 branches.
```

Comment:
would be interesting to have a description of the tests included here and the optional ones too

Reply:
Good point! I wonder if it perhaps is a bit premature, at least for the optional tests? I'm afraid the list would be constantly changing (or worse, outdated).
Another question is if it should be here or in the `test/e2e/README.md`?
What do you think?

Reply:
good point for the optional tests, let's wait a bit for when they're more "stable"
I guess adding the description for the required ones to `test/e2e/README.md` and add a link would work